Data/tree_inventory/README.md

```diff
@@ -0,0 +1,18 @@
+# tree_inventory
+
+This folder contains the `inventory.csv` file that records tree inventory
+data for the TEMPEST plots. Its fields include:
+
+Name          | Description
+------------- | ----------------------------------------
+Plot          | Plot name (string, Control/Fresh/Salt)
+Section       | Plot section (string, East/Center/West)
+Tag           | Tag number (integer)
+Species_code  | Species code (genus + species; https://plants.usda.gov/)
+DBH_{year}    | Diameter at 1.4 m in {year}, cm (numeric)
+Date_{year}   | Date inventory performed in {year} (string, YYYY-MM-DD)
+Status_{year} | Status in {year} (string, LI=live; DS=dead standing; DC=??)
+Sapflux       | If sapflux tree, noted here (string)
```

Comment:
I think it's best if we change this to `Sapflow_ID` across the board. In `ports.csv` it's labeled as `Tree_ID` which is confusing. (I'll create a separate issue for this)

Reply:
FWIW I don't see why `inventory.csv` carries this information at all. Let the sap flux datasets worry about that (i.e. let them have a single table showing the C1 maps to tree xxxx, F2 to xxxx, etc.

Reply:
But for now I will change for consistency